src/collections/_documentation/guides/grouping-and-fingerprints/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Grouping & Fingerprints
+title: Customize Event Grouping
```

Comment:
This seems like an unrelated change.

Reply:
yes..

Reply:
We should not bundle unrelated changes, that's why I highlighted this one :)